src/components/elements/FilterChips.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import Chip from 'material-ui/Chip'
+import ByPath from 'object-path'
+
+export const FilterChips = ({filter, selectSpecificFilter, records, onRequestDelete}) => {
+    let selectedFilters = ByPath.get(filter, selectSpecificFilter)
+    return selectedFilters? selectedFilters.map((selectedFilter) => hydrateChip(selectedFilter, records))
```

Comment:
```
return selectedFilters
    ? selectedFilters
        .map()
        .map()
    : <div></div>
```

is my preferred format for this. Lining up the map statements makes it more clear that it's a pipeline rather than a map called within another map call. Lining up the ternary operator increases uniformity with the rest of the codebase.